firebase-crashlytics/src/main/java/com/google/firebase/crashlytics/internal/model/CrashlyticsReport.java

```diff
@@ -84,6 +84,13 @@ public static Builder builder() {
   @NonNull
   public abstract Session getSession();
 
+  // TODO
```

Comment:
Can we add a comment as to why this is a TODO?

Reply:
Because we haven't implemented NDK yet. I'll add that to the TODO.

Reply:
👍